storage/spanstore/downsampling_writer_benchmark_test.go

```diff
@@ -18,6 +18,9 @@ func (n *noopWriteSpanStore) WriteSpan(span *model.Span) error {
 	return nil
 }
 
+//Benchmark result:
+//BenchmarkDownSamplingWriter_WriteSpan-12 without bytepool 50	  33352181 ns/op	18981065 B/op	  219750 allocs/op
+//BenchmarkDownSamplingWriter_WriteSpan2-12   with bytepool 100	  28070245 ns/op	16137975 B/op	  127494 allocs/op
```

Comment:
What is writespan2 here? is it with the pooling? That's still feels like a lot of allocations, we might want to pprof this. Once we generated the []bytes in the pool, we shouldn't be seeing 0 allocations.

Reply:
Let me try that shortly.